deel-1-flexbox-positioneren/styles/styles.css

```diff
@@ -1 +1,109 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
+
+/*//////////////  OPDRACHT 1///////////////////*/
+.opdracht-1 {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+}
+
+.opdracht-1 ul {
+    /*border: 2px dotted pink;*/
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    flex-basis: 800px;
+}
+
+/*//////////////// OPDRACHT 2  ///////////////////*/
+.opdracht-2 {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+}
+
+.opdracht-2 ul {
+    /*border: 2px dotted deeppink;*/
+    display: flex;
+    flex-direction: row;
+    flex-basis: 800px;
+    justify-content: center;
+
+}
+.opdracht-2 li {
+    /* de totale breedte van de parent (880px) opgedeeld in 3 gelijke stukken*/
+    flex-basis: 267px;
+}
+
+/*///////////////// OPDRACHT 3 //////////////////*/
+.opdracht-3 {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+}
+
+.opdracht-3 ul {
+    /*border: 2px dotted deeppink;*/
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    flex-basis: 100%;
+    height: 100%;
+    gap: 130px;
+
+
+}
+
+.item-top-left {
+    align-self: flex-start;
+}
+
+.item-bottom-right {
+    align-self: flex-end;
+}
+
+/*////////// OPDRACHT 4 /////////*/
+.opdracht-4 {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+}
+
+.opdracht-4 ul {
+    /*border: 2px dotted deeppink;*/
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    flex-wrap: wrap;
+    gap: 12px;
+}
+
+/*///////////// OPDRACHT 5 ///////////*/
+.opdracht-5 div{
+    /*border: 2px solid greenyellow;*/
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    height: 100%;
```

Comment:
De height-property hoeft hier niet te werden gebruikt, indien je in het html-file in voor .opdracht 5 het div-element de class flex-container meegeeft. Deze wordt namelijk in het do-not-edit.css file gebruikt om de height-property aan te passen voor deze opdracht.